src/comm/MAVLinkProtocol.cc

```diff
@@ -191,7 +191,7 @@ void MAVLinkProtocol::_linkStatusChanged(LinkInterface* link, bool connected)
         // Send command to start MAVLink
         // XXX hacky but safe
         // Start NSH
-        const char init[] = {0x0d, 0x0d, 0x0d};
+        const char init[] = {0x0d, 0x0d, 0x0d, 0x0d};
```

Comment:
It's just something small address sanitizer caught.
A few lines after that

    link->writeBytes(init, 4);

I just slipped it in here because it seemed trivial.